.circleci/config.yml

```diff
@@ -32,10 +32,18 @@ jobs:
           key: gradle-{{ checksum "build.gradle.kts" }}
       - store_test_results:
           path: build/test-results
+      - store_test_results:
+          path: domain/timeline/build/test-results
+      - run:
```

Comment:
Storing artifacts in circle was about to get large and potentially messy as each module has 2 folders that we store - the test reports, and the jacoco reports.
When dealing with lots of files (these 4 are just directories; within them there are loads of files), circleci document the use of a tar job which can improve the performance (speed of upload and storage space) of uploading the artifacts.